client/my-sites/marketplace/components/billing-interval-switcher/index.jsx

```diff
@@ -0,0 +1,51 @@
+import { PlansIntervalToggle } from '@automattic/plans-grid/src';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import SelectDropdown from 'calypso/components/select-dropdown';
+
+const Container = styled.div`
+	.plans-interval-toggle {
+		display: inline-flex;
+	}
+`;
+
+const PlansIntervalToggleLabel = styled.span`
+	font-size: 14px;
+	color: var( --studio-gray-60 );
+	margin-right: 10px;
+`;
+
+const BillingIntervalSwitcher = ( { billingPeriod, onChange, compact } ) => {
+	const translate = useTranslate();
+	const monthlyLabel = translate( 'Monthly price' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Monthly Price' )` *ES Score: 12*